juniper/src/integrations/bson.rs

```diff
@@ -36,13 +36,13 @@ where
     S: ScalarValue,
 {
     fn resolve(&self) -> Value {
-        Value::scalar((*self).to_rfc3339())
+        Value::scalar(DateTime::from(*self).to_rfc3339())
     }
```

Comment:
These compilation failures can be avoided by adding the `chrono-0_4` feature flag to the `bson` dependency (this was added in 2.0).